src/app/services/filter.service.ts

```diff
@@ -6,20 +6,27 @@ import { FilterModel } from '../models/filter.model';
 import { HttpClientService } from '../shared/http-module/http.service';
 import { WorkItem } from './../models/work-item';
 
+import {
+  AND, ENCLOUSER, EQUAL, IN,
+  NOT_EQUAL, NOT_IN, OR, P_END,
+  P_START, SUB_STR
+} from './query-keys';
+
 @Injectable()
 export class FilterService {
   public filters: FilterModel[] = [];
   public activeFilters = [];
   public filterChange = new Subject();
   public filterObservable: Subject<any> = new Subject();
 
-  public and_notation = '$AND';
-  public or_notation = '$OR';
-  public equal_notation = '$EQ';
-  public not_equal_notation = '$NE';
-  public in_notation = '$IN';
-  public not_in_notation = '$NIN';
-  public sub_str_notation = '$SUBSTR';
+  public and_notation = AND;
+  public or_notation = OR;
+  public equal_notation = EQUAL;
+  public not_equal_notation = NOT_EQUAL;
+  public in_notation = IN;
+  public not_in_notation = NOT_IN;
+  public sub_str_notation = SUB_STR;
+  public str_enclouser = ENCLOUSER;
```

Comment:
What's the point of assigning these instance values from constants? They don't change, and shouldn't. The code refers to both the constants and the instance vars. You should be able to clean these up.